### PR TITLE
Temporarily reduce regression test runtime

### PR DESCRIPTION
**Pull request by talbring**
I reduced the number of iterations for some of the regression tests. They need ~ 45 min now. This should at least temporarily help.

Along with that I found one or two bugs ...

Nevertheless, all of this shouldn't affect a lot of people, so I hope we can merge this in asap, to avoid any more travis runs to fail :)

### CODE REVIEW

**Comment by rsanfer** [comment]
Hi Tim,
thanks, this is really useful. Let's wait until the test cases finish, but I agree, we should get this in sooner than later. The regression test issue is slowing us all down, definitely something to get fixed.

File: TestCases/serial_regression.py [hunk 1]
```diff
@@ -93,8 +93,8 @@ def main():
     fixedCL_naca0012           = TestCase('fixedcl_naca0012')
     fixedCL_naca0012.cfg_dir   = "fixed_cl/naca0012"
     fixedCL_naca0012.cfg_file  = "inv_NACA0012.cfg"
-    fixedCL_naca0012.test_iter = 500
-    fixedCL_naca0012.test_vals = [-5.856803,-0.469459,0.299943,0.019385] #last 4 columns
+    fixedCL_naca0012.test_iter = 100
```
Comment (economon) [inline TestCases/serial_regression.py]: You may have already checked this, but does 100 iterations cover at least one change of the AoA with fixed Cl mode? Just want to make sure we're covering that.

Comment (talbring) [reply TestCases/serial_regression.py]: I didn't think of this ... but fortunately in the config file the UPDATE_ALPHA is set to 5, so it should be covered.

Comment (economon) [reply TestCases/serial_regression.py]: I think that is actually the number of times to update over the length of the calculation. Can you please double check this? I think we can just add a few iterations to this one if necessary...

Comment (talbring) [reply TestCases/serial_regression.py]: I just checked, the interval when alpha is updated is computed as EXT_ITER/UPDATE_ALPHA in the code.

**Pull request merged**